pytorch_lightning/trainer/trainer.py

```diff
@@ -293,6 +294,13 @@ def __init__(
 
             terminate_on_nan: If set to True, will terminate training (by raising a `ValueError`) at the
                 end of each training batch, if any of the parameters or the loss are NaN or +/-inf.
+
+            auto_scale_batch_size: If set to True, will `initially` run a batch size
+                finder trying to find the largest batch size that fits into memory.
+                The results will be stored in self.hparams.batch_size in the lightning module.
```

Comment:
```suggestion
                The result will be stored in self.hparams.batch_size in the LightningModule.
```
There is only one result, right?